packages/fx-core/src/common/local/portChecker.ts

```diff
@@ -21,19 +21,23 @@ const botDebugPortRegex = /--inspect[\s]*=[\s"']*9239/im;
 const botDebugPorts = [9239];
 const botServicePorts = [3978];
 
-async function detectPortListening(port: number): Promise<boolean> {
+async function detectPortListening(port: number, logger?: LogProvider): Promise<boolean> {
   try {
+    logger?.info(`Start to detect port: ${port}`);
```

Comment:
Can we use `logger.debug` or any way to by-default hide the log?

Reply:
The debug level will always be ignored in vscode. Would you like to send the telemetry instead of showing the log to our users?

Reply:
I see. I'm ok with log or telemetry.